Zukini.UI.Examples.Features/App.config

```diff
@@ -12,8 +12,9 @@
     <add key="MaximizeBrowser" value="true" />
     <add key="ScreenshotDirectory" value="..\..\..\Screenshots" />
     <!-- Application Under Test Settings -->
-    <add key="GoogleUrl" value="http://www.google.com" />
-    <add key="W3SchoolsBaseUrl" value="http://www.w3schools.com" />
+    <add key="GoogleUrl" value="https://www.google.com" />
+    <add key="GoogleHttpUrl" value="http://www.google.com" />
+    <add key="W3SchoolsBaseUrl" value="https://www.w3schools.com" />
     <add key="JsonPlaceholderApiUrl" value="http://jsonplaceholder.typicode.com" />
```

Comment:
The HTTP setting is to verify that WaitForNavigation does timeout (negative test). Should always use HTTPS setting for other tests. Could remove the negative test as an alternative.